src/large_pages/node_large_page.cc

```diff
@@ -30,12 +30,16 @@
 #include "util.h"
 #include "uv.h"
 
+#if defined(__linux__) || defined(__FreeBSD__)
+#include <string.h>
 #if defined(__linux__)
 #ifndef _GNU_SOURCE
 #define _GNU_SOURCE
-#endif
+#endif  // ndef _GNU_SOURCE
```

Comment:
```suggestion
#endif  // ifndef _GNU_SOURCE
```
Also happens in one or two other places, e.g., line 132.